packages/terra-menu/src/_MenuUtils.js

```diff
@@ -0,0 +1,57 @@
+const getPopupHeight = (contentHeight) => {
+  if (contentHeight <= 40) {
```

Comment:
I mean it's still multiples of 40 at that point, but not all multiples of 40 are valid beyond 160.